apps/server/src/modules/task-card/uc/task-card.uc.spec.ts

```diff
@@ -410,7 +452,7 @@ describe('TaskCardUc', () => {
 			await uc.update(user.id, taskCard.id, taskCardUpdateParams);
 			expect(taskService.update).toBeCalledWith(user.id, taskCard.task.id, taskParams);
 		});
-		it('should throw if due date is before visible at date', async () => {
+		it('should throw if due date is before visibleAtDate', async () => {
```

Comment:
either write a business case (due date) or technical case (dueDate) but please dont mix them up in the test naming.

Reply:
Dont know about Virgils opinion on that, but maybe there is some decision from the arc group in https://docs.dbildungscloud.de/pages/viewpage.action?spaceKey=DBH&title=Code+Styles+08.02.23 where you can have a look

Reply:
I have changed all the names in this test file. I have decided to use the original variable names, as the tests are for programmers and the naming should describe what is happening in the code.